Hast.Abstractions/Hast.Vitis.Abstractions/Services/AzureHardwareImplementationComposerBuildProvider.cs

```diff
@@ -0,0 +1,189 @@
+﻿using Hast.Layer;
+using Hast.Synthesis.Abstractions;
+using Hast.Vitis.Abstractions.Extensions;
+using Hast.Vitis.Abstractions.Models;
+using Hast.Xilinx.Abstractions;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Hast.Vitis.Abstractions.Services
+{
+    public class AzureHardwareImplementationComposerBuildProvider : IHardwareImplementationComposerBuildProvider
+    {
+        private const string BlobContainerName = "hastlayer-attestation";
+        private readonly IAzureStorageServiceFactory _azureStorageServiceFactory;
+        private readonly ILogger<AzureHardwareImplementationComposerBuildProvider> _logger;
+
+        public Dictionary<string, BuildProviderShortcut> Shortcuts { get; } = new Dictionary<string, BuildProviderShortcut>();

```

Comment:
```suggestion
        public Dictionary<string, BuildProviderShortcut> Shortcuts { get; } = new();

```

Ez kellene az összes hasonló helyen is.

Reply:
Nem az volt, hogy Támogatjuk a .Net Core 3.1-et ami ezt nem tudja?

Reply:
Mint target framework de, de ehhez az nem kell, csak az SDK.